pyp5js/http/web_app.py

```diff
@@ -66,6 +68,12 @@ def render_sketch_view(sketch_name, static_path):
         elif not content_file.exists():
             return '', 404
 
+        if static_path.endswith(('gif', 'jpg', 'tga', 'png')):
+            img = open(content_file, 'rb').read()
+            return send_file(io.BytesIO(img),
+                             attachment_filename='%s.jpg' % static_path,
+                             mimetype='image/'+static_path[-3:])
```

Comment:
Can you add a unit test to ensure we're serving the file correctly?

Reply:
Edge case test scenario: static file names with upper case =]

If this test does not exist for the JS files, it would be nice to have it tested too.